app/(content)/_layout.js

```diff
@@ -0,0 +1,86 @@
+import { Tabs } from "expo-router/tabs";
+import { useSegments } from "expo-router";

```

Comment:
remove this

Reply:
removed the useSegments import if that's what you meant